src/contrib/mlir/compiler.cpp

```diff
@@ -66,6 +66,12 @@ using llvm::make_unique;
 
 using namespace ngraph::runtime::ngmlir;
 
+/// Enable/Disable loop fusion optimization in affine dialect.
+static llvm::cl::opt<bool>
```

Comment:
Using LLVM cmd line options will not work if running full models (e.g through a framework or nbench). Might be better to introduce a string env variable that you can parse. 
`NGRAPH_MLIR_OPT="loop-fusion=ON;loop-tiling=OFF"`

Reply:
This is similar what we currently do for LLVM options.

Reply:
Renaming and reusing that env var, as discussed. Something like this works now:
NGRAPH_MLIR_OPTIONS="--enable-affine-loop-fusion=false"